backend/grant/task/views.py

```diff
@@ -0,0 +1,15 @@
+from datetime import datetime
+
+from flask import Blueprint, jsonify
+from grant.task.jobs import JOBS
+from grant.task.models import Task, tasks_schema
+
+blueprint = Blueprint("task", __name__, url_prefix="/api/v1/task")
+
+
+@blueprint.route("/", methods=["GET"])
+def task():
+    tasks = Task.query.filter(Task.execute_after <= datetime.now()).all()
+    for each_task in tasks:
+        JOBS[each_task.job_id](each_task.id)
```

Comment:
Each job fn should take in the whole task object, not the id. Saves a DB request to grab it (again.)

Reply:
Each job should probably also be wrapped in a try / exception clause, so that one faulty job doesn't kill the whole lot. Also a good time for sentry logging.

Reply:
👍 Makes sense on all fronts!